src/main/java/RandomNumber.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Random;
+
+public class RandomNumber {
+    final int digit;
```

Comment:
```suggestion
    private final int digit;
```
<br>
final 을 붙였다하더라도 외부에서 직접 접근하지 못하도록 접근제어자를 private 로 선언한 후, 필요한 경우 메서드를 통해서 값을 가져오셔야 합니다.